web/server/codechecker_server/api/report_server.py

```diff
@@ -469,6 +477,20 @@ def extended_data_db_to_api(erd):
         fileId=erd.file_id)
 
 
+def comment_kind_str(status):
```

Comment:
Is there any string involved in this conversion? Just another enum value is returned.
I'm thinking would it be better to tie this conversion functions closer to the CommentKind local enum type?
With some other name which makes the conversion more explicit to_thrift_type, from_thrift_type.

A function docstring would be nice for the conversion functions.